src/dryad/format/endf/ReactionIdentifiers.hpp

```diff
@@ -105,31 +154,34 @@ namespace endf {
     }
 
     /**
-     *  @brief Return whther or not the MT number is for a derived reaction
+     *  @brief Return whether or not the MT number is for a derived reaction
      *
-     *  @param[in] mt   the MT number
+     *  @param[in] material   the ENDF material
+     *  @param[in] mt         the MT number
      */
     static bool isDerived( const ENDFtk::tree::Material&, int mt ) {
 
       return derived_.find( mt ) != derived_.end();
     }
 
     /**
-     *  @brief Return whther or not the MT number is for a summation reaction
+     *  @brief Return whether or not the MT number is for a summation reaction
      *
-     *  @param[in] mt   the MT number
+     *  @param[in] material   the ENDF material
+     *  @param[in] mt         the MT number
      */
     static bool isSummation( const ENDFtk::tree::Material& material, int mt ) {
 
       switch ( mt ) {
 
-        case    16 : return material.hasSection( 3, 875 ) ? true : false;
-        case    18 : return material.hasSection( 3, 19 )  ? true : false;
-        case   103 : return material.hasSection( 3, 600 ) ? true : false;
-        case   104 : return material.hasSection( 3, 650 ) ? true : false;
-        case   105 : return material.hasSection( 3, 700 ) ? true : false;
-        case   106 : return material.hasSection( 3, 750 ) ? true : false;
-        case   107 : return material.hasSection( 3, 800 ) ? true : false;
+        case  16 : return material.hasSection( 3, 875 ) ? true : false;
+        case  18 : return material.hasSection( 3, 19 )  ? true : false;
```

Comment:
Subactinide fission (MF10 MT18) was added a few years ago. The data as put in MF10 and not MF3 for the simple reason that ENDF rules require certains types of information to be present for fission that were not available for subactinide fission. You know, things like nubar, fission energy release components, etc.

I think that it should have been put in MF3 as a cross section and that the rules should have been updated instead of just dumping the data in MF10. These special exceptions make no sense to me. This can be handled like any other reaction. Other notorious offenders of the "we have to do something special for this instead of doing things proeprly" are the electron and photoatomic data.